profiles:
  internal:
    camera_id: 0
    frame_rate: 30.0
    image_width: 1280
    image_height: 720
    fourcc: MJPG
    frame_id: internal_camera_optical_frame
    camera_info_url: "~/.ros/calib/internal.yaml"